bonfire/utils.py

```diff
@@ -390,6 +390,14 @@ def get_dependencies(items, optional=False):
         log.debug("clowdapp '%s' has %s: %s", name, key, list(dependencies))
         deps_for_app[name] = dependencies
 
+    for item in items:
+        name = item.get("metadata", {}).get("name")
+        bonfire_deps = item.get("metadata", {}).get("annotations", {}) \
```

Comment:
You get the metadata multiple times, so you could slim this down a bit by storing that

```
    metadata = item.get("metadata", {})
    name = metadata.get("name")
    bonfire_deps = metadata.get("annotations", {}).get("bonfire.dependencies", "").split(',')
```

Its OK the way it is but food for thought

Reply:
that's a very good point. I'll update it